src/prettytable/prettytable.py

```diff
@@ -1324,7 +1355,9 @@ def get_string(self, **kwargs):
                 tmp.extend(line.split("\n"))
             lines = ["|" + line[1:-1] + "|" for line in tmp]
 
-        return "\n".join(lines)
+        # added reset color code so that it doesn't overflow into other print statements
+
+        return "\n".join(lines) + "\033[0m"
```

Comment:
We don't want to return any resets when there's no theme.

Reply:
Fixed